### ISSUE TITLE
Using %i() For an Empty Array Causes Freeze in config/routes.rb

### ISSUE THREAD

**Issue by ksylvest** [flyerhzm/rails_best_practices#270]
**Freezes:**

``` ruby
# config/routes.rb
Rails.application.routes.draw do
  resource :users, only: %i() do
    # ...
  end
end
```

**Runs:**

``` ruby
# config/routes.rb
Rails.application.routes.draw do
  resource :users, only: [] do
    # ...
  end
end
```

**Comment by eignerchris** [comment]
Also seeing this but with `%w[]`

**Comment by eignerchris** [comment]
Did a little digging. Looks like the analyzer gets stuck in a loop. I stepped through the code using pry and got the analyzer to hang. When I `ctrl` + `c`'d the process, it produced the following backtrace: https://gist.github.com/eignerchris/dfe2602f5dd039c4543b43dc5f03a97d

**Comment by wata727** [comment]
This problem is caused by code_analyzer gem. I fixed this bug. 
https://github.com/flyerhzm/code_analyzer/pull/8
Please wait code_analyzer v0.4.7 releases, then you can fix this problem.

**Comment by flyerhzm** [comment]
I have released code_analyzer 0.4.7, please try again

**Comment by yskkin** [comment]
I encountered this problem with `resources :articles, only: %I[index show]` 
using code_analyzer 0.4.7 and rail_best_practices 1.18.1.
